src/utils/filter-children/filter-children.util.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+const renderChild = (child, callback) => {
+  if (callback) return callback(child);
+  return child;
+};
+
+const defaultFilter = (processedText, processedValue, filterType) => {
+  return processedText[filterType](processedValue);
+};
+
+const filterChildren = ({
```

Comment:
Some documentation about how to use this would be good.

Reply:
done 👍